README.md

```diff
@@ -75,3 +69,22 @@ USER=myuser make image push
 # build and push on a different registry
 IMAGE=dockerhub.io/myuser/plugin:tag make image push
 ```
+
+### Testing in OpenShift
+
+Probably the easiest way to test without dev mode (e.g. to test a pull request) is to use the [operator](https://github.com/netobserv/network-observability-operator/), build and push the image to your own registry account and update the `FlowCollector` CR to update the plugin image.
+
+E.g:
+
+```bash
+IMAGE=quay.io/${USER}/plugin:pr-xx make image push
+
+oc edit FlowCollector cluster
+# Here replace image with the newly created one under .spec.consolePlugin.image
+```
+
+If you had previously used the console with the plugin installed, you may need to restart console pods to clear cache:
```

Comment:
not sure if this is 100% accurate (I don't know exactly what is cached and what isn't), but ... it worked for me :)